test/test_distributions.py

```diff
@@ -913,6 +918,11 @@ def get_sp_dist(jax_dist):
             ]
         ),  # Covariance
     ),
+    T(dist.LowerTruncatedPowerLaw, -jnp.pi, jnp.array([2.0, 5.0, 10.0, 50.0])),
+    T(dist.DoublyTruncatedPowerLaw, -1.0, 1.0, 2.0),
+    T(dist.DoublyTruncatedPowerLaw, jnp.pi, 5.0, 50.0),
+    T(dist.DoublyTruncatedPowerLaw, -1.0, 5.0, 50.0),
+    T(dist.DoublyTruncatedPowerLaw, jnp.pi, 1.0, 2.0),
```

Comment:
could you use np instead of jnp for those global arrays?

Reply:
Resolved in [aba947b](https://github.com/pyro-ppl/numpyro/pull/1807/commits/aba947b30ed46c9d978b05b2ab1f1981e2b0e040).